easybuild/easyconfigs/i/IPython/IPython-5.5.0-CrayGNU-17.08-python2.eb

```diff
@@ -0,0 +1,216 @@
+# Contributed by Theofilos Manitaras (CSCS)
+easyblock = 'Bundle'
+
+name = 'IPython'
+version = '5.5.0'
+versionsuffix = '-python2'
+pyshortver = '2.7'
+
+homepage = 'http://ipython.org/index.html'
+description = """IPython provides a rich architecture for interactive computing 
+with: Powerful interactive shells (terminal and Qt-based). A browser-based 
+notebook with support for code, text, mathematical expressions, inline plots 
+and other rich media. Support for interactive data visualization and use of GUI 
+toolkits. Flexible, embeddable interpreters to load into your own projects.
+Easy to use, high performance tools for parallel computing."""
+
+toolchainopts = {'pic': True, 'verbose' : False }
+
+toolchain = {'name': 'CrayGNU', 'version': '17.08'}
+
+dependencies = [
+    ('cray-python/17.06.1', EXTERNAL_MODULE),
+    ('PyExtensions', pyshortver),
+    ('mock', '2.0.0', versionsuffix),
+    ('PyZMQ', '17.0.0b1', '%s-zmq4' % versionsuffix),
+]
+
+exts_defaultclass = 'PythonPackage'
+
+exts_list = [
+    ('nose', '1.3.7', {
```

Comment:
nose/1.3.7 is part of cray-python but as they both use the same version it should not be an issue.